subscription_manager/controller.py

```diff
@@ -79,25 +79,33 @@ def get_subscription_by_name(self, subscription_name: str) -> Subscription:
         """
         if type(subscription_name) is not str:
             raise WrongTypeException(
-                f"Found wrong field 'subscription_name' type, expected:<str>, received:{type(subscription_name), subscription_name}"
+                f"Found wrong field 'subscription_name' type, expected: <str>, received: {type(subscription_name), subscription_name}"
             )
         if len(subscription_name) < 1:
             raise InvalidValueException(
                 "Subscription name length should be more than one"
             )
-        recieved_subscription: dict = self.dbhelper.get_subscription(subscription_name)
-        recieved_subscription.pop("_id")
-        new_date_type = {"start_date": recieved_subscription["start_date"].date()}
-        recieved_subscription.update(new_date_type)
-        return Subscription(**recieved_subscription)
+        received_subscription: dict = self.dbhelper.get_subscription(subscription_name)
+        received_subscription.update({"start_date": received_subscription["start_date"].date()})
```

Comment:
Can this logic be transferred to "select from db" part ?

Reply:
Done